package.json

```diff
@@ -38,14 +38,17 @@
     "axios": "^0.16.2",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "formidable-charts": "^0.0.7",
+    "history": "^4.7.2",
     "jumpstate": "^2.2.2",
+    "lodash": "^4.17.4",
     "moment": "^2.18.1",
     "react": "^15.6.1",
     "react-addons-shallow-compare": "^15.6.0",
     "react-dates": "^12.2.3",
     "react-dom": "^15.6.1",
     "react-flexbox-grid": "^1.1.3",
     "react-redux": "^5.0.4",
+    "react-router-redux": "next",
```

Comment:
I'd use a caret and the latest version here (so `^4.0.8`), then you're moving the burden of not breaking the app over to the maintainers of `react-router-redux`